pdns/recursordist/aggressive_nsec.cc

```diff
@@ -488,7 +488,7 @@ static void addRecordToRRSet(time_t now, const DNSName& owner, const QType& type
   }
 }
 
-bool AggressiveNSECCache::synthesizeFromNSEC3Wildcard(time_t now, const DNSName& name, const QType& type, std::vector<DNSRecord>& ret, int& res, bool doDNSSEC, ZoneEntry::CacheEntry& nextCloser, const DNSName& wildcardName, const OptLog& log)
```

Comment:
marking this as this might actually  be a bug

Reply:
I thinks both synthesize functions should set `res` to `RCode::NoError` when returning`true`. @rgacogne Agree?